posts/app.py

```diff
@@ -68,13 +68,29 @@ def get_all_posts():
 
 @app.route('/v1/<post_id>', methods=['GET'])
 def get_post_by_id(post_id):
-    """Get the post with the specified ID."""
-    # serialize otherwise nonserializable ObjectIDs
+    """Get the post with the specified ID.
+    """
     post_list = find_posts_in_db(
         app.config["COLLECTION"], post_id=ObjectId(post_id))
     return serialize_posts_to_json(post_list)
 
 
+@app.route('/v1/<post_id>/delete', methods=['DELETE'])
+def delete_post_by_id(post_id):
+    """Delete the post with the specified ID.
+
+    On deletion, only deletes posts matching post_id and with an author_id
+    matching the request parameter's author ID. Assumes that the passed in
+    author_id is authenticated and nonmalicies. I.e. this should only be
```

Comment:
Small typo here - should it be "non-malicious"?

Reply:
Thanks, that's embarassing. Done https://github.com/knative-portability/large-events/pull/52/commits/7497b9b1b9ebe574403ac3a9f5ac0d0eb74aad27